exercises/protein-translation/src/lib.rs

```diff
@@ -1 +1,21 @@
+pub struct CodonsInfo;
 
+impl CodonsInfo {
+    pub fn name_for(&self, codon: &str) -> Result<&'static str, &'static str> {
```

Comment:
Agree that returning `Option` makes more sense here than returning `Result`. Please put the edits which change the return type into a separate commit. This also applies to the return type of `of_rna`.